src/pages/home/TermsUndecorated.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="main m-8 overflow-scroll">
+    <TermsContent />
+    <p>&nbsp;</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { useI18n } from 'vue-i18n';
+import TermsContent from './TermsContent.vue';
+
+export default defineComponent({
+  name: 'TermsUndecorated',
+  components: { TermsContent },
+});
+</script>
+
+<style lang="scss">
+// TODO Use common styles from Home.vue rather than duplicating
+.grid--main {
+  h1 {
+    @apply text-2xl font-bold mt-3;
+  }
+
+  h2 {
+    @apply text-xl font-bold mt-3;
+  }
+
+  h3 {
+    @apply text-lg font-semibold mt-3;
+  }
+
+  h4 {
+    @apply text-base mt-3;
+  }
+
+  h5 {
+    @apply text-sm mt-3;
+  }
+
+  h6 {
+    @apply text-xs mt-3;
+  }
+
+  p {
+    @apply my-2;
+  }
+
+  strong {
+    @apply font-bold;
+  }
+
+  u {
+    @apply underline;
+  }
+
+  em {
+    @apply italic;
+  }
+
+  a {
+    @apply text-primary-dark underline;
+
+    &:hover {
+      text-decoration: none;
+    }
+  }
+}
+
+ol.outline {
+  list-style-type: none;
+  margin-left: 0;
+  counter-reset: item;
+  outline-style: none;
+}
+ol.outline > li {
+  counter-increment: customlistcounter;
+  display: block;
+}
+ol.outline:first-child,
+ol.outline:second-child {
+  counter-reset: customlistcounter;
+}
+ol.outline > li:before {
+  counter-increment: item;
+  /*content: "#" counter(customlistcounter);*/
+  font-weight: bold;
+  float: left;
+  width: 3em;
+}
+ol.one > li:before {
+  content: '1.' counters(item, '.') ' ';
+}
+ol.two > li:before {
+  content: '2.' counters(item, '.') ' ';
+}
+ol.three > li:before {
+  content: '3.' counters(item, '.') ' ';
+}
+ol.four > li:before {
+  content: '4.' counters(item, '.') ' ';
+}
+ol.five > li:before {
+  content: '5.' counters(item, '.') ' ';
+}
+ol.six > li:before {
+  content: '6.' counters(item, '.') ' ';
+}
+ol.seven > li:before {
+  content: '7.' counters(item, '.') ' ';
+}
+ol.eight > li:before {
+  content: '8.' counters(item, '.') ' ';
+}
+ol.nine > li:before {
+  content: '9.' counters(item, '.') ' ';
+}
+ol.ten > li:before {
+  content: '10.' counters(item, '.') ' ';
+}
+ol.eleven > li:before {
+  content: '11.' counters(item, '.') ' ';
+}
+ol.twelve > li:before {
+  content: '12.' counters(item, '.') ' ';
+}
+ol.thirteen > li:before {
+  content: '13.' counters(item, '.') ' ';
+}
+ol.fourteen > li:before {
+  content: '14.' counters(item, '.') ' ';
+}
+ol.fifteen > li:before {
+  content: '15.' counters(item, '.') ' ';
+}
+</style>
```

Comment:
These styles are going to be globally applied whenever this component gets rendered... not sure that is is a good idea?

Reply:
Since you did add a scoped style, Im assuming you had a reason for this @hueachilles ?

Reply:
This component is more a view and less a component. It is not meant to be reused. `TermsContent.vue` should be reused instead of this component.
I copied the styles from `Home.vue` so it will render in a similar manner.

Reply:
I'll scope all styles it for good measure.

Reply:
Actually I can't scope it. It must be "global" for it to apply to child components.

Reply:
@hueachilles will give proper response in a bit, but dropping this: https://vuejs.org/api/sfc-css-features.html#deep-selectors

tho better ways to do this tho